server/drivers/crate/index.js

```diff
@@ -69,6 +67,26 @@ async function runQuery(query, connection) {
   }
 }
 
+/**
+ * Get connection string with options
+ * @param {*} connection
+ */
+function getConnectionString(connection) {
+  let connectionString = connection.ssl ? 'https://' : 'http://';
```

Comment:
Should this check if protocol is supplied in current host field?

I'm wondering if anyone may have included protocol previously, and this would double up on it or not.

If protocol is supported in previous implementation, perhaps host is parsed with URL first? https://nodejs.org/docs/latest-v12.x/api/url.html

Reply:
Thanks for the input!
I reworked it to use URL in general. This should take care of case where a combination of `host`, `port`, `auth` or `protocol` have previously been supplied in the `host` field.

Reply:
Excellent! I wasn't aware of URL myself until not long ago.